site/en/docs/crux/api/index.md

```diff
@@ -0,0 +1,592 @@
+
+# Optional
+# This appears below the title and is an optional teaser
+subhead: >
+  The Chrome UX Report API gives low-latency access to aggregated Real User Metrics (RUM) from the Chrome User Experience Report.
+
+# Required
+date: 2022-06-23
+
+# Optional
+# Include an updated date when you update your post
+# updated: 2020-10-16
+
+# Optional
+# How to add a new author
+# https://developer.chrome.com/docs/handbook/how-to/add-an-author/
+authors:
+  - simonhearne
+
+# Optional
+# How to a new tag
+# https://developer.chrome.com/docs/handbook/how-to/add-a-tag/
+tags:
+  - performance
+---
+
+The Chrome UX Report API gives low-latency access to aggregated Real User Metrics (RUM) from the Chrome User Experience Report.
+
+## Common use case
+
+The Chrome UX Report API allows for the querying of user experience metrics for a specific URI like "Get metrics for the `https://example.com` origin."
+
+## Data model
+
+### Record
+
+A discrete piece of information about a page, or site. A record can have data that is specific for an identifier and for a specific combination of dimensions. A record can contain data for one or more metrics.
+
+### Identifiers
+
+Identifiers specify what records should be looked up. In the Chrome UX Report these identifiers are webpages and websites.
+
+### Origin
+
+When the identifier is an origin all data present for all pages in that origin are aggregated together. For example, say the http://www.example.com origin had pages as laid out by this sitemap:
+
+```text
+http://www.example.com
+http://www.example.com/foo.html
+http://www.example.com/bar.html
+```
+
+This would mean that when querying the Chrome UX Report with the origin set to `http://www.example.com`, data for `http://www.example.com/`, `http://www.example.com/foo.html`, and `http://www.example.com/bar.html` would be returned because those are all pages under that origin.
+
+### URLs
+
+When the identifier is a url only data for that specific url will be returned. Looking again to the `http://www.example.com` origin sitemap:
+
+```text
+http://www.example.com
+http://www.example.com/foo.html
+http://www.example.com/bar.html
+```
+
+If the identifier is set to url with the value of `http://www.example.com/foo.html`, only data for that page will be returned.
+
+### Dimensions
+
+Dimensions identify a specific group of data that a record is being aggregated against, e.g. a form factor of Mobile indicates that the record contains information about loads that took place on a mobile device. Each dimension will have a certain number of values, and implicitly the lack of specifying that dimension will mean that the dimension is aggregated over all values. For example, specifying no form factor indicates that record contains information about loads that took place on any form factor.
+
+#### Form Factor
+
+The device class that the end-user used to navigate to the page. This is a general class of device split into `Phone`, `Tablet`, and `Desktop`. The `formFactor`
```

Comment:
The final sentence just ends mis sentence. Would also put in caps:

```suggestion
The device class that the end-user used to navigate to the page. This is a general class of device split into `PHONE`, `TABLET`, and `DESKTOP`
```

Also is ECT not another dimension? Should it have a paragraph here?